### PR TITLE
made changes to metadata, json, file names, and directory name for en…

### PR DESCRIPTION
**Pull request by philshin-google**
…voy-proxy to more accurate Istio Envoy naming convention

### CODE REVIEW

File: dashboards/istio-envoy/metadata.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 sample_dashboards:
   -
-    category: Envoy
-    id: envoy-proxy-prometheus
-    display_name: Envoy Proxy
+    category: Istio Envoy
```
Comment (EvanSimpson) [inline dashboards/istio-envoy/metadata.yaml:3]:
```suggestion
    category: Istio
```

File: dashboards/istio-envoy/metadata.yaml [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+    id: istio-envoy-prometheus
+    display_name: Istio Envoy
     description: |-
       This dashboard has charts displaying Live Proxies, Average Uptime (minutes), Allocated Memory, Heap Size, Unhealty Proxies, Total Active Connections, Total Requests Rate, Allocated Memory, Upstream Network Traffic, CDS Update Attemps, CDS Update Success, and CDS Update Failure
     related_integrations:
-      - id: envoy
+      - id: istio-envoy
```
Comment (EvanSimpson) [inline dashboards/istio-envoy/metadata.yaml:9]:
```suggestion
      - id: istio
```

File: dashboards/istio-envoy/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
-### Dashboards for Envoy Proxy
+### Dashboards for Istio Envoy Proxy
 
 #### Notes
 
 - This dashboard is based on [metrics collected by Istio](https://istio.io/latest/docs/ops/integrations/prometheus/) for the built-in Envoy sidecar.
 
-|Envoy Proxy|
+|Istio Envoy Proxy|
```
Comment (EvanSimpson) [inline dashboards/istio-envoy/README.md:7]:
```suggestion
|Istio Envoy Prometheus Overview|
```

File: dashboards/istio-envoy/README.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
-### Dashboards for Envoy Proxy
+### Dashboards for Istio Envoy Proxy
```
Comment (EvanSimpson) [inline dashboards/istio-envoy/README.md:1]:
```suggestion
### Dashboards for Istio
```

Comment (EvanSimpson) [inline dashboards/istio-envoy/README.md:1]: Can you also name the parent folder just `istio` - it's possible there will be other istio dashboards for components other than envoy in the future.

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/istio-envoy/metadata.yaml (hunk 2 above)
Comment (philshin-google) [reply dashboards/istio-envoy/metadata.yaml]: done.

File: dashboards/istio-envoy/metadata.yaml (hunk 1 above)
Comment (philshin-google) [reply dashboards/istio-envoy/metadata.yaml]: Done.

File: dashboards/istio-envoy/README.md (hunk 3 above)
Comment (philshin-google) [reply dashboards/istio-envoy/README.md]: done.

File: dashboards/istio-envoy/README.md (hunk 4 above)
Comment (philshin-google) [reply dashboards/istio-envoy/README.md]: done.

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**